recipes-extend/bootloader-flasher-linux/files/bootloader_flash.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/python3
+
+# Imports
+import serial
+import argparse
+import time
+import os
+import zipfile
+import subprocess
+from subprocess import Popen, PIPE, CalledProcessError
+from sys import platform
+import glob
+import shlex
+
+class FlashUtil:
+	def __init__(self):
+		self.__scriptDir = os.getcwd()
+		self.__rootDir = os.path.abspath(os.path.join(self.__scriptDir, '..', '..', '..', '..'))
+		self.__imagesDir = os.path.abspath(os.path.join(self.__rootDir, 'target', 'images'))
+		
+		self.__setupArgumentParser()
+
+		self.__setupSerialPort()
+		self.__writeBootloader()
+
+	# Setup CLI parser
+	def __setupArgumentParser(self):
+		# Create parser
+		self.__parser = argparse.ArgumentParser(description='Utility to flash bootloader on RZ SBC Board.\n', epilog='Example:\n\t./bootloader_flash.py')
+
+		# Add arguments
+		# Serial port arguments
+		self.__parser.add_argument('--serial_port', default='/dev/ttyUSB0', dest='serialPort', action='store', help='Serial port used to talk to board (defaults to: /dev/ttyUSB0).')
+		self.__parser.add_argument('--serial_port_baud', default=115200, dest='baudRate', action='store', type=int, help='Baud rate for serial port (defaults to: 115200).')
+
+		# Images
+		self.__parser.add_argument('--image_writer', default=f'{self.__imagesDir}/Flash_Writer_SCIF_rzg2l-sbc.mot', dest='flashWriterImage', action='store', type=str, help="Path to Flash Writer image (defaults to: <path/to/your/package>/target/images/Flash_Writer_SCIF_rzg2l-sbc.mot).")
```

Comment:
Please verify is that `Flash_Writer_SCIF_rzg2l-sbc.mot` or `Flash_Writer_SCIF_RZG2L_SBC.mot`.

Then adjust the build script accordingly if its incorrect: https://github.com/Renesas-SST/rz-build-scripts/pull/12/files#diff-aaacb7961dac99f6bda142ff596cdf01d5b39788a89e989496125c739904aa9bR667